lib/pages/chat_list/chat_list.dart

```diff
@@ -410,12 +411,13 @@ class ChatListController extends State<ChatList>
             await TomBootstrapDialog(
               client: client,
               recoveryWords: recoveryWords,
-            ).show(context);
+            ).show(TwakeApp.routerKey.currentContext ?? context);
```

Comment:
Why do I have to use `TwakeApp.routerKey.currentContext ?? context`. In which case is `context` wrong?

Reply:
because the ToomBootstrapDialog need to put in the center of the screen, so I have to use the global context (which is for the biggest Navigator), the current `context` is the context for navigator inside ChatList.

Reply:
fixed to be more understandable